Cann't install nightly build

npm install --save-dev parcel@2.0.0-nightly.504

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@parcel%2ffs-search - Not found
npm ERR! 404
npm ERR! 404  '@parcel/fs-search@2.0.0-beta.1' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\***********\AppData\Local\npm-cache\_logs\2021-01-03T10_47_53_977Z-debug.log

--- a/lerna.json
+++ b/lerna.json
@@ -1,4 +1,4 @@
 {
-  "packages": ["packages/*/*"],
+  "packages": ["packages/*/*", "native-packages/*/*"],
   "version": "independent"
 }

--- a/lerna.json
+++ b/lerna.json
@@ -1,4 +1,4 @@
 {
-  "packages": ["packages/*/*", "native-packages/*/*"],
+  "packages": ["packages/*/*", "native-packages/*"],
   "version": "independent"
 }

